.github/workflows/preview-comment.yml

```diff
@@ -0,0 +1,31 @@
+name: Comment on PR
+
+on:
+  pull_request:
+    branches:
+      - trunk
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ env.BOT_TOKEN }}
+          GIT_AUTHOR_NAME: ${{ env.BOT_NAME }}[bot]
+          GIT_AUTHOR_EMAIL: ${{ env.BOT_NAME }}[bot]@users.noreply.github.com
+          GIT_COMMITTER_NAME: ${{ env.BOT_NAME }}[bot]
+        with:
+          github-token: ${{ env.BOT_TOKEN }}
+          script: |
+            const previewLink = `https://playground.wordpress.net/wordpress.html?pr=${context.issue.number}`;
```

Comment:
Can we add a bit more context to this? If I am someone opening a PR for the first time and have no idea what this links to or what Playground even is, I should have a brief explanation in the comment and links out to some resources to learn more.

Reply:
Absolutely! It’s only brief because I focused on auto-posting which doesn’t work yet. Let’s flesh out this PR a bit more, then

Reply:
How does the updated version work for you @desrosj ?

Reply:
I've made some additional adjustments. Please have a look and see!